### ISSUE TITLE
Several crashes happen since upgrading from 2.2.3 to 2.2.6 related to PortingLib

### ISSUE THREAD

**Issue by MindStudioOfficial** [MehVahdJukaar/FarmersDelightRefabricated#122]
### Minecraft version

1.21.1

### Farmer's Delight version

2.2.6

### Fabric Loader version

0.16.9

### Fabric API version

0.110.0+1.21.1

### Description

Game Crashes when Right-Clicking with axe:

Also reported here https://github.com/Fabricators-of-Create/Porting-Lib/issues/152

```
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1838.method_8045()" because "context" is null
	at knot//io.github.fabricators_of_create.porting_lib.tool.events.BlockToolModificationEvent.<init>(BlockToolModificationEvent.java:38)
	at knot//io.github.fabricators_of_create.porting_lib.util.PortingHooks.onToolUse(PortingHooks.java:73)
	at knot//net.minecraft.class_1743.wrapOperation$zzi000$porting_lib_base$onStripToolAction(class_1743.java:1035)
```

Full Crash report https://pastebin.com/6MtXsiTE

Error didn't occur in 2.2.3 and happens after upgrading to 2.2.6 possibly related to the changes in portinglib dependancy

Update: 
also checked 2.2.4 - no crash
also checked 2.2.5 - crashes

Farmers Delight is the only mod that uses PortingLib in our modlist (according to the dependancy tree)

### Steps to reproduce

1. get any axe
2. right click any block

### Mod list

Contained within the crash report

https://pastebin.com/6MtXsiTE

### Logs

https://pastebin.com/6MtXsiTE

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [X] I am using performance or shader mods

**Comment by ChrysanthCow** [comment]
Mhm. This was one of the fixes we implemented back when we explicitly used our own builds. Ill fix soon.

**Comment by ChrysanthCow** [comment]
Fixed in 2.2.7
